src/test/java/core/basesyntax/constants/Constants.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.constants;
+
+public class Constants {
+    public static final String CORRECT_PATH_TO_READ = "src/main/resources/validFileForTest.csv";
+    public static final String INCORRECT_PATH_TO_READ = "src/main/resources/nonExistentFile.csv";
```

Comment:
if you are talking about these lines, they are here because they are used in tests. The package and class of constants are located in the test package

Reply:
you've placed test resources under main folder. The path should be src/test/resources/validFileForTest.csv